src/WormholeRouter.sol

```diff
@@ -135,9 +142,9 @@ contract WormholeRouter {
      */
     function settle(bytes32 targetDomain, uint256 batchedDaiToFlush) external {
         bytes32 sourceDomain = domains[msg.sender];
-        require(sourceDomain != bytes32(0), "WormholeRouter/sender-not-gateway");
+        if (sourceDomain == bytes32(0)) revert SenderNotDomain(msg.sender, sourceDomain);
         address gateway = gateways[targetDomain];
-        require(gateway != address(0), "WormholeRouter/unsupported-target-domain");
+        if (gateway == address(0)) revert UnsupportedTargetDomain(gateway);
```

Comment:
Similar to above this should also report the `targetDomain` and not the `gateway`.